GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,87 @@ influence the speed of HP attrition and if it outweighs the benefits
+However, you should remember that the outfit is limited to any 10 (ten)
+items only. On the contrary, if the item is not required directly (ranged
+weapon, medkits, ampoules with drugs, food, or else), it may be stored in
+the shared inventory stash.
+
+The shared inventory stash is of unlimited size. It has a physical location on
+the map and may be looted. So, it is wise to keep it safe or change 
+its location depending on the current situation. It would be a pity to lose 
+valuables or powerful projectiles by not taking care of the shared inventory. 
+
+To relocate the shared inventory stash, first, you need to remove ALL 
+of the items from it anywhere you want to make it empty. Then, any stashed
+item (or a pile of items) will automatically become the new stash. The process 
+takes a couple of turns, so you should plan shared inventory stash relocation
+in advance.
+You should keep your eye on the item management stat in the character menu
+`#`. If this stat is too low (which may be caused by an injury or low 
+calmness level), it will be difficult for the character to work with items in 
+the character's personal outfit or in the shared inventory stash.
+
+To open the pointman's outfit, press `O` (if you want to see the pointman's
+personal inventory outfit and manipulate items stored there) or `i` (if
+you want to stash something in the shared inventory from the pointman's
+outfit).
+To open the shared inventory stash, press `I`.
+To open the whole team's belongings (personal and stashed), press `T`.
+To manage items on the ground (trigger or fling it directly not putting
+it into any inventory to save time), press `G`.
+
+To pick up an item from the ground to the pointman's outfit, press `g`
+or `,`. Items that have some direct use (melee weapon or armor) 
+will be equipped in the personal equipment outfit while if it has no 
+direct use or the equippment outfit is full, the item will be moved
+to the shared inventory.
+
+When you are managing items in the inventory or on the ground, you should
+choose the item with the LMB or arrows and `RET` buttons to get the item
+management menu. You may want to choose all items from the particular 
+location by pressing `*`.
+
+You may want to cycle among the characters' inventory outfits without leaving 
+the inventory menu. To do that, you may press `<` or `>` to check different 
+inventories: character's outfit, stashed inventory, objects on the ground, etc. 
+
+Quite often, you will need to modify your environment somehow. There are
+numerous ways how to do that, so try and experiment!
+You may craft items, change the terrain, cook food, upgrade or change items,
+and do many other things!
+
+To craft or modify items, you need some basic components and a workshop
+station or any other terrain that may be used to modify items (a pool
+of water, a source of fire, etc). Stand next to the workshop station, drop
+the components, stand on them, press `M`, and point to the direction
```

Comment:
The "stand on them" is a bit misleading. The character is already standing on them, right?

"point in the direction"